src/ec/suite_b/ops/p256.rs

```diff
@@ -297,6 +298,23 @@ fn p256_scalar_inv_to_mont(a: &Scalar<Unencoded>) -> Scalar<R> {
     acc
 }
 
+const N_N0: N0 = N0([0xccd1c8aa, 0xee00bc4f]);
+fn p256_scalar_mul_mont(r: &mut [Limb], a: &[Limb], b: &[Limb]) {
+    limbs_mont_mul(r, a, b, &N_N0)
+}
+
+fn p256_scalar_sqr_rep_mont(r: &mut [Limb], a: &[Limb], rep: Limb) {
+    debug_assert!(rep >= 1);
+    p256_scalar_mul_mont(r, a, a);
+    for i in 1..rep {
+        p256_scalar_mul_mont(r, r, r);
+    }
+}
```

Comment:
Doing this would regress the performance on x86-64 and AArch64 targets that have an optimized implementation of this.